cds_ils/importer/providers/cds/rules/values_mapping.py

```diff
@@ -40,6 +40,13 @@
     "migration": ["migration"],
 }
 
+MEDIUMS = {
```

Comment:
Other possible values not added (yet):  "digital","film","optical disk", "internet", "online","u-matic","ebook","betacam","D3","Streaming video","DVCAM","U-matic SP"

Reply:
does it mean the lit here is OK?

Reply:
Yes